src/index.js

```diff
@@ -23,33 +23,44 @@ import {
 
 import { initCSSPolyfill } from "./scroll-timeline-css"
 
-initCSSPolyfill();
+function initPolyfill() {
+  // initCSSPolyfill returns true iff the host browser supports SDA
+  if (initCSSPolyfill()) {
+    return true;
```

Comment:
Nit: Return type is ignored and will return undefined if not early exiting.  "return;" instead of "return true;".